modules/snappy/1.2.1/source.json

```diff
@@ -0,0 +1,10 @@
+{
+    "url": "https://github.com/google/snappy/archive/refs/tags/1.2.1.tar.gz",
```

Comment:
>  BcrValidationResult.FAILED: snappy@1.2.1 is using an unstable source url: `https://github.com/google/snappy/archive/refs/tags/1.2.1.tar.gz`.

Snappy does not seem to have created download artifacts for its release. This matches the other URLs for snappy.